### DISCUSSION TITLE
freezing PICO_W works, but fails with PICO, what am i missing?

### DISCUSSION THREAD

**Discussion by TLmv** [micropython/micropython#12018] [RP2040 / Pico] [answered]
I just froze a few lettertypes into V1.2 for the PICO_W by using a manifest file

```
include("$(PORT_DIR)/boards/manifest.py")

require("bundle-networking")
freeze("$(MPY_DIR)/gui/fonts", "arial12.py")
freeze("$(MPY_DIR)/gui/fonts", "arial35.py")
```
this worked out, but i wanted it for the regular pico as well.
so i copied the V1.2 for the PICO to the build folder and removed the one for the pico_w, copied the manifest.py file to the boards/PICO folder and did the same as for the pico_w:

```
make BOARD=PICO submodules
make BOARD=PICO clean
make BOARD=PICO
```

it appeared to do make without errors. however, this time the fonts were not included as with the pico_w build and you do not see them in the frozen_mpy, so i removed `require("bundle-networking")` but that did not solve it...furthermore, it seems to have taken not v1.2 but v1.9 but i do not understand where that comes from since i placed the v1.2 in the build folder.

is there something i forgot to change before i can make a build of the regular PICO?

**Comment by robert-hh** [comment, marked as answer]
You have to add the lines:
```
# Board specific version of the frozen manifest
set(MICROPY_FROZEN_MANIFEST ${MICROPY_BOARD_DIR}/manifest.py)
```
to mpconfigboard.cmake in the PICO directory to tell the build system to start with the board's manifest.py.

**Comment by TLmv** [comment]
thank you, i had forgotten that i did that a long time back for the PICO_W. 
although it still made a V1.19 instead of the v1.2 i placed in the build folder with the fonts this at least works...i'll sort out later how to update to 1.2 before freeze.
